### ISSUE TITLE
tpacpi-bat cause T470s' fun runs at maximum speed after resuming from suspend to ram

### ISSUE THREAD

**Issue by cnzhx** [teleshoes/tpacpi-bat#77]
As stated in the title, the CPU fan in T470s with i7-7600U will run at a high speed after resuming from suspend when tpacpi-bat is installed while the service is not enabled using systemctl. This problem disappears after removing its tpacpi config in `/etc/conf.d/tpacpi` and service file in `/usr/lib/systemd/system/tpacpi.service` (this service has not been enabled though systemctl).

P.S. Does TLP use tpacpi-bat as a component?

**Comment by teleshoes** [comment]
1) tlp does use tpacpi-bat, yes
2) the only thing this project addresses is the tpacpi-bat perl script, period. the services/etc were written by other folks who really wanted a place to host them. i do not use them personally, and i do not provide support for them.
3) if you find a bug in tpacpi-bat itself, please report it.

**Comment by linrunner** [comment]
tpacpi.service doesn't run on suspend/resume. tlp-sleep.service does, but doesn't call tpacpi-bat. 

Conclusion: a connection to your problem is highly improbable.

**Comment by cnzhx** [comment]
Thank you teleshoes and linrunner for the clarifications. And very sorry for my mistake on jumping to this conclusion.

Maybe I jumped too rush on the conclusion. It's probably the problem just kicked in when I was testing with installing and uninstalling tpacpi-bat. I should've test more times for sure.

But as you explained, it's unlikely the problem is caused by the using of tpacpi-bat. And I ran again into the same problem after I cleaned the install of tpacpi-bat, although only once in several times I tested. It looks like the problem won't happen every time after resuming from suspend.

Thanks again.
